packages/Webkul/Shop/src/Http/Controllers/CartController.php

```diff
@@ -115,6 +127,27 @@ public function remove($itemId)
         return redirect()->back();
     }
 
+    /**
+     * Re add products to the cart according to order id
+     *
+     * @return \Illuminate\Http\Response
+    */
+    public function reorder($id)
+    {
+        $order = $this->orderRepository->findOneWhere([
+            'customer_id' => auth()->guard('customer')->user()->id,
+            'id'          => $id,
+        ]);
+
+        foreach ($order->items as $item) {
+            $slugOrPath = strtolower(str_replace(" ","-",$item->name));
```

Comment:
Use space after and before with binary operator

Reply:
done